### PR TITLE
fix(js)!: use patched ref-napi/ffi-napi for nodejs

### PR DESCRIPTION
**Pull request by genaris**
Similar approach as https://github.com/hyperledger/anoncreds-rs/pull/231 to make an easier set up in Node.js environment.

### CODE REVIEW

**Review by TimoGlastra — DISMISSED** [review]

File: wrappers/javascript/package.json [hunk 1]
```diff
@@ -46,6 +46,6 @@
     "typescript": "~4.9.4"
   },
   "engines": {
-    "node": ">= 12"
+    "node": ">= 18"
```
Comment (berendsliedrecht) [inline wrappers/javascript/package.json:49]: I don't think that the root workspace package.json engine influences anything when a user installs `@hyperledger/indy-vdr-nodejs`. I think it should be done in the `indy-vdr-nodejs` package only as well. The shared and react native packages can be used with lower node versions.

Comment (genaris) [reply wrappers/javascript/package.json:49]: Right, actually the restriction will come from the dependencies (patched `ref-napi` and `ffi-napi`) when attempting to do `yarn install`. Anyway, I'll move it to `indy-vdr-nodejs` to make it clear from there.

**Review by TimoGlastra — APPROVED** [review]

**Comment by TimoGlastra** [comment]
Do we want to release a 0.2.x for indy-vdr that only supports nodejs 18 right away?

We should probably also add a version compatiblity table to the readme: https://github.com/hyperledger/aries-askar/tree/main/wrappers/javascript/aries-askar-nodejs#version-compatibility

**Review by andrewwhitehead — APPROVED** [review]

**Comment by andrewwhitehead** [comment]
@TimoGlastra We need to get a 0.4 release out of the Python wrapper as well, at the moment there's still no non-dev arm64 build. Is there anything else missing?

**Pull request merged**